src/http/get-media-000image/index.js

```diff
@@ -26,11 +26,25 @@ exports.handler = response(async req => {
   if (!originalImage) {
     return reply.text('Not Found', { statusCode: 404 });
   }
+  // set up optimization params
```

Comment:
lets only do this logic if resize is set.  I think you also need to call optimiz.resize

Reply:
(specifically filename)